boards/arm/nrf9160_innblue22/nrf9160_innblue22_defconfig

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2020 InnBlue
+# SPDX-License-Identifier: Apache-2.0
+
+# SOC / Board
+CONFIG_SOC_SERIES_NRF91X=y
+CONFIG_SOC_NRF9160_SICA=y
+CONFIG_BOARD_NRF9160_INNBLUE22=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# Enable TrustZone-M
+CONFIG_ARM_TRUSTZONE_M=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# Enable uart driver
+CONFIG_SERIAL=y
+CONFIG_UART_NRFX=y
```

Comment:
This is redundant. `UART_NRFX` is by default enabled for all nRF SoCs, when `SERIAL` is enabled.
Same story in `boards/arm/nrf9160_innblue22/nrf9160_innblue22ns_defconfig`.

Reply:
thank you, good point!